packages/web-frontend/src/screens/LandingPage/useNavigation.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useDispatch } from 'react-redux';
+import { setProject, setRequestingAccess } from '../../projects/actions';
+import { REQUEST_PROJECT_ACCESS, LANDING } from '../../containers/OverlayDiv/constants';
+import { attemptUserLogout, setOverlayComponent } from '../../actions';
+
+export const useNavigation = () => {
```

Comment:
I'm doing my best to work with the existing redux navigation while still making it easy to refactor later which is why these are in their seperate util file. Once we refactor to proper routes none of these utils will be necessary.

Reply:
Nice work with these, seems like a good way to make a simple use of what we already have.